tests/modules/nf-scil/denoising/mppca/nextflow.config

```diff
@@ -0,0 +1,8 @@
+process {
+
+    publishDir = { "${params.outdir}/${task.process.tokenize(':')[-1].tokenize('_')[0].toLowerCase()}" }
+
+    withNAME: "DENOISING_MPPCA" {
+        extent = 7
```

Comment:
Here, to accelerate the tests, try an extent of 3. If it doesn't work, use 5 (the default).

Reply:
it works, but it's not really faster